src/inmanta/server/config.py

```diff
@@ -63,6 +78,27 @@
 #############################
 # server
 #############################
+server_bind_address = Option(
+    "server", "bind-address", "127.0.0.1", "A list of addresses on which the server will listen for connections", is_list
+)
+server_bind_port = Option("server", "bind-port", 8888, "The port on which the server will listen for connections", is_int)
+
+
+def get_bind_port() -> int:
+    if Config.is_set("server", "bind-port") or Config.is_set("server", "bind-address"):
+        # Use new bind-port option
+        if Config.is_set("server_rest_transport", "port"):
+            LOGGER.warning(
```

Comment:
Lets use warnings for this and push them also to the logs
https://docs.python.org/3.6/library/warnings.html
https://docs.python.org/3.6/library/logging.html#logging.captureWarnings